mmf/models/unit/unit_base_model.py

```diff
@@ -0,0 +1,426 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# Mostly copy-pasted from
+# https://github.com/facebookresearch/detr/blob/master/models/detr.py
+import torch
+import torch.nn.functional as F
+from mmf.utils import box_ops
+from mmf.utils.distributed import get_world_size, is_dist_initialized
+from torch import nn
+
+from .backbone import build_backbone
+from .matcher import build_matcher
+from .misc import NestedTensor
+from .transformer import build_transformer
+
+
+ATTRIBUTE_CLASS_NUM = 401
+MAX_ATTR_NUM = 16
```

Comment:
Suggestion : Should we make these configurable?

Reply:
(added to config)